oracle/README.md

```diff
@@ -59,56 +59,56 @@ The project can be started by executing the following steps:
+| ------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| API_URL            | The URL the node uses to fetch the latest price for the asset tracked by the oracle. This oracle node relies on an external 3rd-party service to get price information to provide to the oracle contract. We do not endorse this service neither are we affiliated with them in any way. We only use the service for demonstration purposes. If you wish to run the node you can sign-up for a free api key [here](https://www.cryptocompare.com/). If you wish to use another pricing api service feel free to replace `API_URL` entirely. |
+| ORACLE_CONTRACT_ID | Deterministic contract id of the oracle contract deployed in step 5.                                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
+| WALLET_SECRET      | Private key of the first deterministic wallet provided by the [fuels-rs](https://github.com/FuelLabs/fuels-rs) sdk. This private key correspondes to the `owner` address specified in the oracle contract's [`Forc.toml`](./project/contracts/oracle-contract/Forc.toml). This address is also configured in step 4 to have the maximum amount of the [BASE_ASSET](https://github.com/FuelLabs/sway/blob/master/sway-lib-std/src/constants.sw).                                                                                             |
+| FUEL_PROVIDER_URL  | Fuel-core network url normally set as http://localhost:4000/graphql for development.                                                                                                                                                                                                                                                                                                                                                                                                                                                        |
```

Comment:
In terms of the preview it looks exactly the same.
Now we have a bunch of new characters to visually format the table in non-preview mode.

From 1 perspective this is useless, from another it helps a little.
I'm somewhat ambivalent regarding this change so your call.

Reply:
I didn't do this manually, not sure what formatter made the change. I don't think it needs to be changed.